### PR TITLE
docs: add security policy

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR adds a security policy for how we handle vulnerabilities in code-server.

See ["Adding a security policy to your repository"](https://docs.github.com/en/code-security/security-advisories/adding-a-security-policy-to-your-repository) for more information.

### CODE REVIEW

File: docs/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+Only the latest version of `code-server` is currently support with security updates. These security updates will be patched in the version after the current version.
```
Comment (jsjoeio) [inline docs/SECURITY.md:5]:
```suggestion
Only the latest version of code-server is currently support with security updates. These security updates will be patched in the version after the current version.
```

File: docs/SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+Only the latest version of code-server is currently support with security updates. These security updates will be patched in the version after the current version.
```
Comment (jawnsy) [inline docs/SECURITY.md:5]:
```suggestion
Coder sponsors development and maintenance of the code-server project. We will fix security issues within 90 days of receiving a report, and publish the fix in a subsequent release. The code-server project does not provide backports or patch releases for security issues at this time.
```

**Review by jawnsy — APPROVED** [review]
LGTM!

File: docs/SECURITY.md (hunk 2 above)
Comment (jsjoeio) [reply docs/SECURITY.md]: WAY better. It's like you've done this before ♥️

**Pull request merged**